app/views/steps/income/manage_without_income/edit.html.erb

```diff
@@ -0,0 +1,25 @@
+<% title t('.page_title') %>
+<% step_header(path: crime_applications_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+  <span class="govuk-caption-xl"><%= t('steps.income.caption') %></span>
+    <%= govuk_error_summary(@form_object) %>
+    <%= step_form @form_object do |f| %>
+        <%= f.govuk_radio_buttons_fieldset(:manage_without_income, legend: { tag: 'h1', size: 'xl' }) do %>
+          <% @form_object.choices.each_with_index do |choice, index| %>
+            <% if choice == ManageWithoutIncomeType::OTHER %>
+              <%= f.govuk_radio_button :manage_without_income, choice.value do %>
+                <% f.govuk_text_area :manage_other_details,
+                                     label: { size: 's' } %>
```

Comment:
If you remove the size attribute, the label text will display without the bold styling
```suggestion
                <% f.govuk_text_area :manage_other_details  %>
```

Reply:
You're a genius. Literally trying to find this solution for my next page (ngl - didn't realise it was wrong on this page until I started the new one)! Thank you!